apps/sms/views/shared/test/unit/navigation_test.js

```diff
@@ -657,6 +657,8 @@ suite('navigation >', function() {
 
       window.Navigation = NavigationFactory(fakeWindow);
       navigator.mozHasPendingMessage.withArgs('notification').returns(false);
+
+      Navigation.setReady();
```

Comment:
> What we need is to have a test that's failing before your patch and working with your patch. so it should be possible to fake document.readyState to interactive, call toPanel or init, and do a .then(done, done) at the end (without sending "load" event but calling setReady). This should fail on master and work with your patch.

> What do you think ?

The only thing I'm concerned about is that such test may by a little bit confusing when we read it in the future,  since I've removed code that was relying on ```document.readyState``` or ```load``` event so we're going to mock something that doesn't have any effect :) But if you think it's fine, I can do it!